src/main/java/org/openhab/binding/zwave/handler/ZWaveThingHandler.java

```diff
@@ -1202,6 +1222,44 @@ public void handleCommand(ChannelUID channelUID, Command command) {
         }
     }
 
+    @Nullable
+    Command convertCommandToDataType(ChannelUID channelUID, DataType channelDataType, Command command,
+                                     DataType dataType) {
+
+        if (!(command instanceof State)) {
+            logger.debug("NODE {}: Command of data-type {} received for channel {} that must be converted to " +
```

Comment:
Could you look at shortening this debug message? It's a debug level, so doesn't really need to be so long. 

```
            logger.debug("NODE {}: Command of type {} received for channel {} to be converted to type {}, is not a State",
```

Reply:
Thanks for the feedback! In general I always spam debug logs, but you are right.. for this binding many people will have debug activated and it already produces a lot log messages.

I shortened the debug messages, is it now fine for you or did I removed too much information? :smile:

Reply:
Thanks - looks good to me.